Demo/Common/Networking/PaymentsRequest.swift

```diff
@@ -47,7 +47,6 @@ internal struct PaymentsRequest: APIRequest {
         try container.encodeIfPresent(data.socialSecurityNumber, forKey: .socialSecurityNumber)
         try container.encode(Locale.current.identifier, forKey: .shopperLocale)
         try container.encodeIfPresent(data.browserInfo, forKey: .browserInfo)
-        try container.encodeIfPresent(data.checkoutAttemptId, forKey: .checkoutAttemptId)
```

Comment:
No need to send `checkoutAttemptId` ?

Reply:
yes, the whole point of the PR is send the `checkoutAttemptId ` in the payment method details instead of the root of the payments call